tests/credentials_tests.rb

```diff
@@ -95,10 +96,27 @@
         aws_secret_access_key: 'dummysecret',
         aws_session_token: 'dummytoken',
         region: 'us-west-1',
+        STS_ENDPOINT: "https://sts.amazonaws.com"
         aws_credentials_expire_at: expires_at
       ) { Fog::AWS::Compute.fetch_credentials(use_iam_profile: true) }
     end
 
+    ENV["AWS_STS_REGIONAL_ENDPOINTS"] = "regional"
+    ENV["AWS_REGION"] = "us-west-1"
```

Comment:
I think, given the rest of the implementation details, that this should maybe use the `AWS_DEFAULT_REGION` key?

Reply:
Good catch, I will get this oversight fixed.